src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+public class Ui {
+
+    public static void identity() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        horizontalLine();
+        System.out.println("Hello! I'm Snow!");
+        System.out.println("What can I do for you?");
+        horizontalLine();
+    }
+
+    public static void printBye() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    public static void horizontalLine() {
+        System.out.println("____________________________________________________________");
+    }
```

Comment:
Perhaps having this as a constant instead of a function would work better? Also, I could not find this being used elsewhere.